packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.spec.tsx

```diff
@@ -0,0 +1,128 @@
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+import {
+  screen,
+  renderComponent,
+  waitFor,
+  fireEvent,
+} from '../../../test-utils';
+import CustomViewLoader, { type TCustomView } from './custom-view-loader';
+
+const mockShowNotification = jest.fn();
+
+jest.mock('@commercetools-frontend/sentry');
+jest.mock('@commercetools-frontend/actions-global', () => ({
+  useShowNotification: () => mockShowNotification,
+}));
```

Comment:
If we want the notification to work, we need to setup Redux for the test and I thought it would be easier if we mock it.

Do you prefer us to actually render the notification?

Reply:
We have the `renderAppWithRedux`. Is it possible to use that?

Reply:
Ah right I forgot about that. Ok, then it's fine for now.